_format/1.1/index.md

```diff
@@ -1778,8 +1816,456 @@ If a server encounters a query parameter that does not follow the naming
+the following rule applies by default:
+
+  1. If the unrecognized value is a JSON object, and the only thing that makes 
+     it unrecognized is that it contains one or more keys that have no meaning
+     assigned to them (in the latest revision of the profile that the application 
+     is aware of), then the application **MUST** simply ignore those unknown 
+     keys and continue processing the profile.
+
+  2. However, in all other cases, the application **MUST** assume that the 
+     profile has been applied erroneously and **MUST** totally ignore the 
+     profile (i.e., process the document as if the profile were not there).
+
+In the case of our example [timestamps profile], it does not define its own 
+rules, so the above defaults would apply. 
+
+Under the first of these default rules, the unrecognized value we saw 
+above (with the `createdUnixEpoch` key) would be processed as though the 
+`createdUnixEpoch` key simply weren't present, and the application would still 
+be able to use the data in the `created` key. 
+
+However, if the user instead provided the following value, the whole timestamps
+profile would need to be ignored:
+
+```json
+{
+  //...
+  "timestamps": { 
+    "modified": "Wed Aug 29 2018 15:00:05 GMT-0400", 
+    "created": "2018-08-29T18:38:17.567Z" 
+  }
+}
+```
+
+Ignoring the profile in this case is required by the second default rule, 
+because the value for the `modified` key is not recognized under the profile's
+requirement that the `modified` key hold a string of the form produced by 
+`JSON.stringify`.
+
+### <a href="#profiles-authoring" id="profiles-authoring" class="headerlink"></a> Authoring Profiles
+
+A profile **MAY** assign meaning to elements of the document structure whose use
+is left up to each implementation, such as resource fields or members of meta
+objects. A profile **MUST NOT** define/assign a meaning to document members 
+in areas of the document reserved for future use by the JSON API specification. 
+
+For example, it would be illegal for a profile to define a new key in a 
+document's [top-level][top level] object, or in a [links object], as JSON API 
+implementations are not allowed to add custom keys in those areas.
+
+Likewise, a profile **MAY** assign a meaning to query parameters or parameter 
+values whose details are left up to each implementation, such as `filter` and 
+all those that parameters that contain a non a-z character. However, profiles 
+**MUST NOT** assign a meaning to query parameters that [are reserved](#query-parameters).
+
+The meaning of an element or query parameter defined by a profile **MUST NOT** 
+vary based on the presence or absence of other profiles.
+
+The scope of a profile **MUST** be clearly delineated. The elements and query 
+parameters specified by a profile, and their meanings, **MUST NOT** change over
+time or else the profile **MUST** be considered a new profile with a new URI.
```

Comment:
@ethanresnick - do you think it is clear enough that objects defined by a profile can expand to include new members over time?

Reply:
This is covered in the section below - perhaps that's sufficient without adding another phrase here.

Reply:
Yeah, I think that's sufficient. Happy to add expand though if you have specific wording in mind.